recipes/holodeck-gw/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "holodeck-gw" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holodeck-gw-{{ version }}.tar.gz
+  sha256: 6a20222953caaf22a98488a0de60a9cc74f97cc4ef912450ece4c5d02ff6a135
+
+build:
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=40.6.0
+    - wheel
+    - cython <3.0.0
+    - numpy <2.0.0
+    - scipy
+    - numba
+    - pip
+    - mpi4py
+  run:
+    - python
+    - kalepy
+    - astropy
+    - cosmopy
+    - h5py
+    - matplotlib-base
+    - scipy
+    - pydoe
+    - psutil
+    - ipywidgets
+    - tqdm
+    - cython <3.0.0
+    - schwimmbad
+    - emcee
+    - george
+    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('numba') }}
+    - {{ pin_compatible('mpi4py') }}
```

Comment:
I'm not sure about those. @isuruf if you have some time can you check this here?

Reply:
There were issues at build-time, and pinning the versions of the above packages fixed the issues.

Reply:
It would be better to understand what is going  on before pinning blindly. The numpy one is probably needed. But maybe the other two can be dropped of you adjust the min numpy version as I showed in the first review.

Reply:
You mean the `pip check` test will fail, not the build, right? If so it is preferred to remove cython and comment out the `pip check` test to avoid polluting user env with build tools.

PS: one can have cython as a build tool only without a pyproct.toml.

Reply:
Can you try to remove the numba and mpi4py here so I can take a look at the error?

Reply:
I've removed the pins and the checks have passed. I believe the previous issues were with the Windows builds, which I've since disabled because the upstream code doesn't currently support Windows.

Reply:
BTW, next time don't close and re-open the PR. It puts an extra burden on the reviewers b/c most of our actions here were already review comments in the previous one that ended up duplicating the effort for everybody.